_blogposts/compiler/a-story-of-lazy-encoding.mdx

```diff
@@ -0,0 +1,122 @@
+---
+author: hongbo
+date: "2020-05-15 18:00" 
+previewImg:
+category: compiler
+title: New Lazy Encoding in BuckleScript
+description: |
+  Highlights of our newest changes to the internal representation of exceptions
+  and how it will provide better stacktraces to our users.
+canonical: https://bucklescript.github.io/blog/2020/05/06/exception-encoding
```

Comment:
Remove this, or change it to the canonical url on the bucklescript blog, if this post is planned to be published on both platforms